build/package/pact-proxy/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/dumb-init /bin/sh
 set -e
-
+ip -4 route list match 0/0 | awk '{print $3 "host.docker.internal"}' >> /etc/hosts
```

Comment:
This change was required to be able to build image in M1 Macs. Docker application in our m1 machines does not allow to modify `/etc/hosts` directory.